### DISCUSSION TITLE
Nextjs build issue in vercel

### DISCUSSION THREAD

**Discussion by chourasiapawanvahak** [wojtekmaj/react-pdf#1620] [Q&A]
I am trying to show a PFD preview on my web page, using the exact same code provided in the sample/next_pages in my nextjs project but facing the below build issue in vercel.

> 

Error: /lib64/libz.so.1: version `ZLIB_1.2.9' not found (required by /vercel/path0/node_modules/canvas/build/Release/libpng16.so.16)
--
19:20:16.083 | at Module._extensions..node (node:internal/modules/cjs/loader:1340:18)
19:20:16.083 | at Module.load (node:internal/modules/cjs/loader:1119:32)
19:20:16.083 | at Module._load (node:internal/modules/cjs/loader:960:12)
19:20:16.083 | at Module.require (node:internal/modules/cjs/loader:1143:19)
19:20:16.083 | at require (node:internal/modules/cjs/helpers:121:18)
19:20:16.084 | at Object.<anonymous> (/vercel/path0/node_modules/canvas/lib/bindings.js:3:18)
19:20:16.084 | at Module._compile (node:internal/modules/cjs/loader:1256:14)
19:20:16.084 | at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
19:20:16.084 | at Module.load (node:internal/modules/cjs/loader:1119:32)
19:20:16.084 | at Module._load (node:internal/modules/cjs/loader:960:12) {
19:20:16.084 | code: 'ERR_DLOPEN_FAILED'

Node: 18
React: 17.0.2
Next: 12.2
I tried deploying sample/next_pages project with the same react and next version in my vercel and it is working perfectly fine but not sure what is the issue with mine. 
Please help.

**Comment by wojtekmaj** [comment]
Try adding these lines:

https://github.com/wojtekmaj/react-pdf/blob/main/sample/next-app/next.config.js#L4-L8
